recognition/include/pcl/recognition/impl/linemod/line_rgbd.hpp

```diff
@@ -167,59 +168,52 @@ pcl::LineRGBD<PointXYZT, PointRGBT>::computeBoundingBox (int i)
   bb.x = bb.y = bb.z = std::numeric_limits<float>::max ();
   bb.width = bb.height = bb.depth = 0.0f;
 
-  float center_x = 0.0f;
-  float center_y = 0.0f;
-  float center_z = 0.0f;
-  float min_x = std::numeric_limits<float>::max ();
-  float min_y = std::numeric_limits<float>::max ();
-  float min_z = std::numeric_limits<float>::max ();
-  float max_x = -std::numeric_limits<float>::max ();
-  float max_y = -std::numeric_limits<float>::max ();
-  float max_z = -std::numeric_limits<float>::max ();
+  Eigen::Vector3f geometric_center = Eigen::Vector3f::Zero ();
+  Eigen::Vector3f min_pos, max_pos;
+  min_pos.fill (std::numeric_limits<float>::max ());
+  max_pos.fill (std::numeric_limits<float>::lowest ());
   std::size_t counter = 0;
   for (std::size_t j = 0; j < template_point_cloud.size (); ++j) 
   {
     const PointXYZRGBA & p = template_point_cloud.points[j];
 
-    if (!std::isfinite (p.x) || !std::isfinite (p.y) || !std::isfinite (p.z))
+    if (!isFinite(p))
```

Comment:
isFinite will be deprecated soon in favor of specific functions, like isXYZFinite